src/reducers/authReducer.ts

```diff
@@ -1,40 +1,54 @@
+import {Dispatch} from 'redux';
+
 import {Constants} from '../actions/types';
 import {LoginActionTypes} from '../actions/loginActions';
+import {logIn, logOut, setAuth} from '../actions/loginActions';
+import {authentication} from '../services/storage/authentication';
 
 export interface AuthState {
-  isLoggedIn: boolean;
-  email: string;
-  password: string;
+  auth: boolean;
 }
 
 const initialState: AuthState = {
-  isLoggedIn: false,
-  email: '',
-  password: '',
+  auth: false,
 };
 
 export const authReducer = (state = initialState, action: LoginActionTypes) => {
   switch (action.type) {
-    case Constants.ON_CHANGE_EMAIL:
+    case Constants.LOG_IN:
       return {
         ...state,
-        email: action.payload,
+        auth: true,
       };
-    case Constants.ON_CHANGE_PASSWORD:
+    case Constants.LOG_OUT:
       return {
         ...state,
-        password: action.payload,
+        auth: false,
       };
-    case Constants.LOG_IN:
+    case Constants.SET_AUTH:
       return {
         ...state,
-        isLoggedIn: true,
-      };
-    case Constants.LOG_OUT:
-      return {
-        state,
+        auth: action.data,
       };
     default:
       return state;
   }
 };
+
+export const tryLogin =
+  (credentials: {email: string; password: string}) => (dispatch: Dispatch) => {
+    const result =
+      (credentials.email === 'email@email.com' &&
+        credentials.password === '12345678') ||
+      false;
```

Comment:
dont do this || false 
If (credentials.email === 'email@email.com' && credentials.password === '12345678') <- this condition is enough to check credentials ...